extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -163,9 +143,8 @@ private function prepareWikiVariables() {
 			);
 		}
 
-		$wikiVariables['navigation2016'] = $navigation;
+		$wikiVariables['localNav'] = $navigation;
```

Comment:
There is nothing we have to rush here with. 
I think we should release Mercury changes first
then app

so next tuesday release mercury/mobile-wiki
and thursday release these changes